src/Dockerfile

```diff
@@ -0,0 +1,103 @@
+# Docker image for extending MoveIt Pro with a custom overlay.
+#
+# Example build command (with defaults):
+#
+# docker build -f ./Dockerfile .
+#
+
+# Specify the MoveIt Pro release to build on top of.
+ARG MOVEIT_STUDIO_BASE_IMAGE
+ARG USERNAME=studio-user
+ARG USER_UID=1000
+ARG USER_GID=1000
+
+##################################################
+# Starting from the specified MoveIt Pro release #
+##################################################
+# The image tag is specified in the argument itself.
+# hadolint ignore=DL3006
+FROM ${MOVEIT_STUDIO_BASE_IMAGE} as base
+
+# Create a non-root user
+ARG USERNAME
+ARG USER_UID
+ARG USER_GID
+
+# Copy source code from the workspace's ROS 2 packages to a workspace inside the container
+ARG USER_WS=/home/${USERNAME}/user_ws
+ENV USER_WS=${USER_WS}
+RUN mkdir -p ${USER_WS}/src ${USER_WS}/build ${USER_WS}/install ${USER_WS}/log
+COPY ./src ${USER_WS}/src
+
+# Also mkdir with user permission directories which will be mounted later to avoid docker creating them as root
+WORKDIR $USER_WS
+# hadolint ignore=DL3008
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    groupadd --gid $USER_GID ${USERNAME} && \
+    useradd --uid $USER_UID --gid $USER_GID --shell /bin/bash --create-home ${USERNAME} && \
+    apt-get update && \
+    apt-get install -q -y --no-install-recommends sudo && \
+    echo ${USERNAME} ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/${USERNAME} && \
+    chmod 0440 /etc/sudoers.d/${USERNAME} && \
+    cp -r /etc/skel/. /home/${USERNAME} && \
+    mkdir -p \
+      /home/${USERNAME}/.ccache \
+      /home/${USERNAME}/.config \
+      /home/${USERNAME}/.ignition \
+      /home/${USERNAME}/.colcon \
+      /home/${USERNAME}/.ros && \
+    chown -R $USER_UID:$USER_GID /home/${USERNAME} /opt/overlay_ws/
+
```

Comment:
I think you need these for the camera to work
```suggestion

# Install additional Kortex Vision dependencies
# NOTE: The /opt/overlay_ws folder contains MoveIt Pro binary packages and the source file.
# hadolint ignore=SC1091
RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
    --mount=type=cache,target=/var/lib/apt,sharing=locked \
    . /opt/overlay_ws/install/setup.sh && \
    apt-get update && \
    apt-get install -y \
      gstreamer1.0-tools \
      gstreamer1.0-libav \
      libgstreamer1.0-dev \
      libgstreamer-plugins-base1.0-dev \
      libgstreamer-plugins-good1.0-dev \
      gstreamer1.0-plugins-good \
      gstreamer1.0-plugins-base
      
```

Reply:
Good catch. I was testing with `picknik_ut_moveit_studio_ws`, which already has this. But I guess it needs to be here too.